_components/search-bar.md

```diff
@@ -4,9 +4,59 @@ type: component
 title: Search Bar
 ---
 
-<div class="preview">
-  <!-- Add HTML markup for example here -->
-  <img src="{{ site.baseurl }}/assets/img/static/Search_Bar_UI_v1.png">
+<div class="preview preview-search-bar">
+
+  <div class="usa-grid-box">
+    <div class="usa-width-one-half">
+      <form action="#" method="post" class="usa-search usa-search-big">           
```

Comment:
We didn't put actions and methods for any of our other forms right? I think we should stay consistent and do it for all of them or none of them.